webui/react/src/components/CompareHyperparameters.tsx

```diff
@@ -23,6 +22,7 @@ import CompareParallelCoordinates from './CompareParallelCoordinates';
 import CompareScatterPlots from './CompareScatterPlots';
 import css from './HpParallelCoordinates.module.scss';
 
+export const NO_DATA_MESSAGE = 'No data available.';
```

Comment:
**super nit:** but generally prefer the const to be defined below interfaces and types